tests/unit/module_utils/net_map/test_networking_definitions_network.py

```diff
@@ -794,3 +794,41 @@ def test_network_definition_parse_valid_router_ok():
     assert router_def.name == "testing-router"
     assert router_def.external_network == "provider-net"
     assert router_def.networks == ["test-net1", "test-net2"]
+
+
+# This will test if passing a route that doesn't match the network version
+# will cause a failure.
+def test_network_definition_parse_tools_route_version_fail():
+    net_name = "testing-net"
+
+    for network_ip in (
+        net_map_stub_data.NETWORK_1_IPV4_NET,
+        net_map_stub_data.NETWORK_1_IPV6_NET,
+    ):
+        net_config = {
+            "network": str(network_ip),
+            "tools": {
+                "multus": {
+                    "ranges": [
+                        {
+                            "start": str(network_ip[100]),
+                            "length": 20,
+                        },
+                    ],
+                    "routes": [
+                        {
+                            "destination": "192.168.121.0/24",
+                            "gateway": "192.168.122.1/32",
```

Comment:
The gateway should be just the IP address and not include the mask. This causes an ipam parser error.

Reply:
Ah mate, I checked this one when I was implementing. It seemed like it wanted the mask: https://github.com/containernetworking/plugins/blob/acf8ddc8e1128e6f68a34f7fe91122afeb1fa93d/plugins/ipam/static/static_test.go#L59

I will check with you on the error, sorry about that one!

Reply:
Oh `s/destination/gateway/` got it!

Reply:
This should be resolved now in the latest revision so resolving